develop-docs/development/testing.mdx

```diff
@@ -135,6 +135,23 @@ If the test runner detects that you used the Django `TestCase` class but you did
 end up needing it, it will yell at you. This protects you against other
 developers yelling at you later for slowing down CI.
 
+### Database tests on temporary models
```

Comment:
We could add a link to an example test file here to use as an example, e.g. https://github.com/getsentry/sentry/blob/774af7dc287a1f9199ce855953c9e9184d8bb9c5/tests/sentry/db/models/fields/test_jsonfield.py

Reply:
Good suggestion, I have added the link to the example